src/Nncase.Core/Utilities/ReplaceUtility.cs

```diff
@@ -115,6 +115,61 @@ public static Call ReplaceParams(Call call, params (ParameterInfo, Expr)[] posAn
         return call with { Parameters = ReplaceMulti(call.Parameters, posAndValue) };
     }
 
+    /// <summary>
+    /// find the old input in old args and replace it with new_input.
+    /// </summary>
+    /// <param name="oldParams">matched old args.</param>
+    /// <param name="target">matched old input.</param>
+    /// <param name="newParam">created new_input.</param>
+    /// <returns>new args list.</returns>
+    /// <exception cref="InvalidOperationException"></exception>
+    public static List<Expr> ReplaceParams(IReadOnlyList<Expr> oldParams, Expr target, Expr newParam)
```

Comment:
The name is not appropriate, here just replace the value in the Expr array to target, and not replace the Params

Reply:
now use list, target, value

Reply:
method name also need to change